dsew_community_profile/delphi_dsew_community_profile/pull.py

```diff
@@ -546,3 +576,109 @@ def generate_prop_signal(df, geo, geo_mapper):
     df.drop(["population", geo], axis=1, inplace=True)
 
     return df
+
+def unify_testing_sigs(positivity_df, volume_df):
+    """
+    Drop any observations with a sample size of 5 or less. Generate standard errors.
+
+    This combines test positivity and testing volume into a single signal,
+    where testing volume *from the same spreadsheet/publish date* (NOT the
+    same reported date) is used as the sample size for test positivity.
```

Comment:
Delphi's been using "reported date" as a soft synonym for publish date, so we should probably use "reference date" here
```suggestion
    same reference date) is used as the sample size for test positivity.
```